README.md

```diff
@@ -1,67 +1,135 @@
+* [**Configuration Guide**](https://www.spokestack.io/docs/Android/turnkey-configuration)
+* [**API Reference**](https://www.javadoc.io/doc/io.spokestack/spokestack-android/latest/index.html)
+* [**Tutorials**](https://www.spokestack.io/blog/tag/tutorial)
+
+
+## Installation
+
+Add the following to your app's `build.gradle`:
+
+```groovy
+android {
+
+    // ...
+
+    compileOptions {
+          sourceCompatibility JavaVersion.VERSION_1_8
+          targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
 
-This example uses a pre-built profile to create a speech recognition pipeline triggered by VAD
-that uses the Google Speech API for speech recognition. The `google-credentials` parameter should
-be the contents of a Google Cloud service account credentials file, in JSON
-format. For more information, see the [documentation](https://cloud.google.com/speech/docs/streaming-recognize).
-See the [javadoc](https://javadoc.io/doc/io.spokestack/spokestack-android) for
-other component-specific configuration parameters.
-
-### Wakeword Detection
-```java
-SpeechPipeline pipeline = new SpeechPipeline.Builder()
-    .useProfile("io.spokestack.spokestack.profile.TFWakewordGoogleASR")
-    .setProperty("wake-filter-path", "<tensorflow-lite-filter-path>")
-    .setProperty("wake-encode-path", "<tensorflow-lite-encode-path>")
-    .setProperty("wake-detect-path", "<tensorflow-lite-detect-path>")
-    .setProperty("wake-threshold", 0.85)
-    .setProperty("google-credentials", "<google-credentials>")
-    .setProperty("locale", "en-US")
-    .build();
+// ...
+
+dependencies {
+    // ...
+
+    implementation 'io.spokestack:spokestack-android:9.1.0'
+
+    // for TensorFlow Lite-powered wakeword detection and nlu, add this one too
+    implementation 'org.tensorflow:tensorflow-lite:1.14.0'
+
+    // for automatic playback of TTS audio
+    implementation 'androidx.lifecycle:lifecycle-common-java8:2.1.0'
+    implementation 'androidx.media:media:1.1.0'
+    implementation 'com.google.android.exoplayer:exoplayer-core:2.11.0'
+}
 ```
 
-This example creates a wakeword-triggered pipeline with the Google Speech
-recognizer. The wakeword trigger uses three trained
-[TensorFlow Lite](https://www.tensorflow.org/lite/) models: a *filter* model
-for spectrum preprocessing, an autoregressive encoder *encode* model, and a
-*detect* decoder model for keyword classification. For more information on
-the wakeword detector and its configuration parameters, click
-[here](https://github.com/spokestack/spokestack-android/wiki/wakeword).
+## Usage
+
+See the [quickstart guide](https://www.spokestack.io/docs/Android/getting-started) for more information, but here's the 30-second version of setup:
+
+1. You'll need to list the `INTERNET` permission in your manifest and request the `RECORD_AUDIO` permission at runtime. See our [skeleton project](https://github.com/spokestack/android-skeleton) for an example of this.
```

Comment:
This is good, but as i have found with ios & rn, a readme quickstart is doomed to bitrot. If the site's quick start isn't quick enough that you want a quicker version, add this to another site doc section and link to it.

Reply:
You could say that about all readme content—the current readme doesn't have a quickstart, and it was completely bitrotten.

The impetus here isn't so much that the site version isn't quick enough, but that some visitors to the repo might not want to bother with an extra click. With a few repos I've come across recently, I've found myself wondering "why won't you just tell me how to _use_ it?" I want the readme to be a foot in the door for those people—try it, like it, go to the site to find out more about it.